openreview/tools.py

```diff
@@ -65,6 +64,28 @@ def create_profile(client, email, first, last, middle = None, allow_duplicates =
     else:
         return profile
 
+def create_profile_ref(client, existing_profile, new_profile_content, source_id, debug=False):
+    # creates new profile with only the new content
+    profile = openreview.Profile(referent=existing_profile.id,
+                                 invitation=existing_profile.invitation,
+                                 signatures=[source_id],
+                                 writers=[source_id],
+                                 content=new_profile_content)
+    try:
+        profile = client.update_profile(profile)
+        if debug:
+            # test if fails to get original
+            db_profile = client.get_profile(id)
+    except Exception as e:
```

Comment:
you should use openreview.OpenReviewException here instead of just "Exception". it's not great to except just anything